kartothek/io/testing/read.py

```diff
@@ -81,7 +81,7 @@ def use_dataset_factory(request, dates_as_object):
 
 def _strip_unused_categoricals(df):
     for col in df.columns:
-        if pd.api.types.is_categorical(df[col]):
+        if pd.api.types.is_categorical_dtype(df[col]):
```

Comment:
Is `is_categorical_dtype` included in pandas `0.23.0`?

Reply:
@marco-neumann-by see: https://github.com/pandas-dev/pandas/blob/v0.23.0/pandas/core/dtypes/common.py#L513

Reply:
Thanks. `master` is currently broken due to a sphinx bug. I'll need to isolate + report + avoid (likely by constraining sphinx) the bug, but might not have time today. I'll take care of this PR here afterwards, so there is nothing to do from your side.

Reply:
@marco-neumann-by should we add this older-yet-still-supported pandas version to our test matrix?

Reply:
Probably after we've reduced our set of pyarrow versions a bit.